src/main/java/no/nav/fo/veilarbregistrering/registrering/scheduler/PubliseringAvRegistreringEventsScheduler.java

```diff
@@ -0,0 +1,47 @@
+package no.nav.fo.veilarbregistrering.registrering.scheduler;
+
+import net.javacrumbs.shedlock.core.SchedulerLock;
+import no.nav.fo.veilarbregistrering.registrering.bruker.PubliseringAvEventsService;
+import no.nav.sbl.featuretoggle.unleash.UnleashService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import static no.nav.fo.veilarbregistrering.log.CallId.leggTilCallId;
+
+@Component
+public class PubliseringAvRegistreringEventsScheduler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PubliseringAvRegistreringEventsScheduler.class);
+
+    private final UnleashService unleashService;
+    private final PubliseringAvEventsService publiseringAvEventsService;
+
+    public PubliseringAvRegistreringEventsScheduler(UnleashService unleashService, PubliseringAvEventsService publiseringAvEventsService) {
+        this.unleashService = unleashService;
+        this.publiseringAvEventsService = publiseringAvEventsService;
+    }
+
+    @Scheduled(cron = "0/10 * * * * *")
+    @SchedulerLock(name = "eventPublisering")
```

Comment:
Ønsker vi denne? Denne baserer seg på ShedLock som vil fungere, men da sørger vi for å bare ha én aktiv node?

Det ble snakket om Optimistic locking i databasen - et annet alternativ er Leader Election fra nais/kubernetes